src/ansys/fluent/core/services/api_upgrade.py

```diff
@@ -44,4 +44,4 @@ def __exit__(self, exc_type, exc_value, exc_tb) -> None:
                 print(
                     "The following solver settings object method could also be used to execute the above command:"
                 )
-                print(f"<solver_session>.{journal_str[len('solver.'):]}")
+                print(f"<solver_session>.settings.{journal_str[len('solver.'):]}")
```

Comment:
@hpohekar, I think  we need to check the version info here. This is applicable for v251 +. 
cc. @mkundu1

Reply:
Done. Thank you.